### ISSUE TITLE
Static section titles spelled out in front-matter(?)

### ISSUE THREAD

**Issue by peterbe** [mdn/stumptown-content#180]
Spawned from [this thread/idea](https://github.com/mdn/stumptown-content/pull/178/files#r337023272). As of [soon](https://github.com/mdn/stumptown-content/pull/178), the heading words `Examples` and `Browser Compatibility` will be hardcoded deep in the `stumptown-content/scripts/build-json/*.js` which isn't a very good place for localizers to type up their translation such as `Exemples` or `Compatibilité des navigateurs`.

One idea would be to extend the front-matter to include these words. E.g. the `en-US` Markdown file could have this diff:

```diff
-browser_compatibility: html.elements.video
+browser_compatibility: 
+    title: Compatibilité des navigateurs
+    query: html.elements.video
```

Then, if a French translator copies that (e.g.) `en-US/some/path/to/page.md` to `fr/some/path/to/page.md` then when they start translating the `title` in the front-matter they can translate all titles in the front-matter.

**Comment by wbamberg** [comment]
I don't think we should do this. It means authors have to duplicate this string (and others) in every single .md file. It means (presumably) that if someone makes a mistake (like "Browser Compatibility", or "Example") then the title won't be consistent.

One of the main goals of structured content is that authors don't have to supply all this meta-content manually, and manually ensure that it's all consistent, because we can rely on the system to do so.

So if we do want to supply this kind of content in stumptown-content, then it should be done inside build-json.

Also, this whole question of "where do these strings get added" goes deeper than just H2 headings. At the moment there are various places where content like this gets added, and we ought to have a consistent plan for the whole thing.
